modules/tinymce/src/core/test/ts/browser/dom/SelectionQuirksTest.ts

```diff
@@ -4,6 +4,8 @@ import { TinyAssertions, TinyContentActions, TinyDom, TinyHooks, TinySelections
 import { assert } from 'chai';
 
 import Editor from 'tinymce/core/api/Editor';
+import Env from 'tinymce/core/api/Env';
+import { VK } from 'tinymce/core/api/PublicApi';
```

Comment:
This should be importing from VK directly instead of from the `PublicApi` that just exports everything.
```suggestion
import VK from 'tinymce/core/api/util/VK';
```